app/views/courses/TeacherClassesView.coffee

```diff
@@ -258,6 +264,7 @@ module.exports = class TeacherClassesView extends RootView
 
   onLoaded: ->
     helper.calculateDots(@classrooms, @courses, @courseInstances)
+    helper.calculateDots(@classrooms, @courses, @sharedCourseInstances)
```

Comment:
does it not override classroom dots calculated above? How about calling this function with an array containing both type of courseInstances

Reply:
good point. let me check.

Reply:
ok it don't override the classroom dots. acutally we don't have `classroom dots` we only have `courseInstance dots`, this `calculateDots` function add stats into third param, the `@sharedCourseInstances` so it works good